python/src/deltachat/testplugin.py

```diff
@@ -282,10 +282,6 @@ def get_online_config(self, pre_generated_key=True, quiet=False):
             if "e2ee_enabled" not in configdict:
                 configdict["e2ee_enabled"] = "1"
 
-            # Enable strict certificate checks for online accounts
-            configdict["imap_certificate_checks"] = str(const.DC_CERTCK_STRICT)
-            configdict["smtp_certificate_checks"] = str(const.DC_CERTCK_STRICT)
```

Comment:
i'd prefer to hide this behind an option, and to let our CI use that "--strict-tls" option.  You can see higher up in testplugin.py how to add an option and query it here via `if pytestconfig.getoption("--strict-tls"):`